pkg/kn/commands/version.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+const (
+	Version = "0.0.0"
```

Comment:
How is this version set ? Imo this should go into the build script via `-ldflags "-X commands.Version=0.1.0"` to `go build`.

Reply:
Yeah, we can do that. I'll add it. 

FYI... Still trying to get the CLA thingie done.